ui/app/AppLayouts/Chat/CommunityComponents/CommunityUserList.qml

```diff
@@ -27,6 +27,31 @@ Item {
 
     property QtObject community: chatsModel.communities.activeCommunity
 
+    onCommunityChanged: {
+        proxyModel.clear()
+        for (let r = 0; r < community.members.rowCount(); r++) {
+            const pubKey = community.members.rowData(r, "address")
+            const nickname = appMain.getUserNickname(pubKey)
+            const identicon = community.members.rowData(r, "identicon")
+            const ensName = community.members.rowData(r, "ensName")
+            const name = !ensName.endsWith(".eth") && !!nickname ? nickname : Utils.removeStatusEns(ensName)
+            const statusType = chatsModel.communities.activeCommunity.memberStatus(pubKey)
+            const lastSeen = chatsModel.communities.activeCommunity.memberLastSeen(pubKey)
+            const lastSeenMinutesAgo = (currentTime / 1000 - parseInt(lastSeen)) / 60
+            const online = (pubKey === profileModel.profile.pubKey) || (lastSeenMinutesAgo < 7)
```

Comment:
Is it correct to assume we are always online? even if we want to be not visible?

Reply:
This logic is borrowed from User.qml, and I assumed if it was done this way, then this is expected behavior.